### PR TITLE
Resolution fixes for HRRR-CAR and HRRR-HI

### PR DESCRIPTION
**Pull request by Brian-Jamison**
HRRR-Hawaii, and the subdomains for HRRR-Caribbean (also added in this PR), needed settings for higher resolution, since some smaller islands were not being plotted.

### CODE REVIEW

File: create_graphics.py [hunk 1]
```diff
@@ -426,6 +426,7 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
         airport_fn=AIRPORTS,
         ax=ax,
         grid_info=field.grid_info,
+        model_name=cla.model_name,
```
Comment (christinaholtNOAA) [inline create_graphics.py:429]: The `cla.model_name` is not used in this fashion. It is meant to be whatever string you might like to put in your map title. Apologies that I didn't catch that in PR #103, but when we are requiring a specific string to enable a feature for a given model, we must be in control of that string. A user could run this with "Experiment 2 HRRR-HI" and it wouldn't work appropriately.

The way we've done it for other models is set the `model` key in the relevant image list, which ends up stored in `cla.images[0]`.

File: adb_graphics/figures/maps.py [hunk 2]
```diff
@@ -73,7 +75,7 @@ class Map():
                         TILE_DEFS dictionary
     '''
 
-    def __init__(self, airport_fn, ax, **kwargs):
+    def __init__(self, airport_fn, ax, model_name=None, **kwargs):
```
Comment (christinaholtNOAA) [inline adb_graphics/figures/maps.py:78]: It is likely best to include this as a keyword argument if it's not required. An entry in the documentation block above would also be helpful.

**Review by christinaholtNOAA — CHANGES_REQUESTED** [review]

**Review by cshartsough — APPROVED** [review]
good job with the fixes and overall work.

**Review by christinaholtNOAA — APPROVED** [review]
Thanks for the super quick response to my earlier comments!

**Comment by Brian-Jamison** [comment]
Thanks!  I forgot to pylint and pytest last time, so hopefully I pass the checks this time.

**Pull request merged**